jerry-core/config.h

```diff
@@ -39,6 +39,7 @@
 # define CONFIG_DISABLE_ES2015_ARROW_FUNCTION
 # define CONFIG_DISABLE_ES2015_BUILTIN
 # define CONFIG_DISABLE_ES2015_CLASS
+# define CONFIG_DISABLE_ES2015_DATAVIEW_BUILTIN
```

Comment:
Please update the `jerry-core/profiles/README.md` file with this new define.

Reply:
This haven't been fixed yet.

Reply:
You can find the new define's description [here](https://github.com/pando-project/jerryscript/pull/2804/files#diff-52116e59bc8cbc847a29fc0adf4a41f3R34).